androidgcs/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-	package="org.dronin.androidgcs" android:versionCode="20170717"
-	android:versionName="Neat">
+	package="org.dronin.androidgcs" android:versionCode="20170826"
+	android:versionName="Neat.1">
```

Comment:
Should decouple this from the main versions because a real Neat.1 might exist later, perhaps -1 suffix instead (which is commonly used for small repackaging type things).

Reply:
good point-- done.